presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/JsonRenderer.java

```diff
@@ -59,17 +58,15 @@ private JsonRenderedNode renderJson(PlanRepresentation plan, NodeRepresentation
     {
         private final String id;
         private final String name;
-        private final String type;
         private final String identifier;
         private final String details;
         private final List<JsonRenderedNode> children;
         private final List<String> remoteSources;
 
-        public JsonRenderedNode(String id, String name, String type, String identifier, String details, List<JsonRenderedNode> children, List<String> remoteSources)
+        public JsonRenderedNode(String id, String name, String identifier, String details, List<JsonRenderedNode> children, List<String> remoteSources)
```

Comment:
IIRC StageDetail L384 uses this field. Did you test the web UI?

Reply:
I don't think so... I think that's using the type from planFragment root plan node.  The type was never serialized to json. If it's supposed to be I can add a getter instead of removing the property, but for now I believe it's unused.